tests/db/common.py

```diff
@@ -18,7 +18,103 @@
 
     Tool to populate the database and dump it for database update testing
 """
+import asyncio
+import importlib
+import os
+import re
+from typing import List
+
+import asyncpg
+
+from inmanta.data.schema import DBSchema
 
 # The content of this file was moved to inmanta.db.util to allow it to be used from other extensions.
 # This import statement is present to ensure backwards compatibility.
 from inmanta.db.util import MODE_READ_COMMAND, MODE_READ_INPUT, AsyncSingleton, PGRestore  # noqa: F401
+
+PACKAGE_NAME = "versions_to_compact"
+
+
+async def compact_and_dump(
+    compact_dir: str, original_dir: str, schema_name: str, database_name: str, postgres_db: asyncpg.Connection
```

Comment:
The `postgres_db` argument cannot be of type `asyncpg.Connection`. This method creates a connection using this object.